src/graphql/v1/generator/fhir_xml_schema_parser.py

```diff
@@ -757,12 +757,10 @@ def generate_properties_for_class(
                         fhir_name=property_name,
                         name=FhirXmlSchemaParser.fix_graphql_keywords(property_name),
                         type_=property_type,
-                        cleaned_type=cleaned_type
-                        if cleaned_type not in FhirXmlSchemaParser.cleaned_type_mapping
-                        else FhirXmlSchemaParser.cleaned_type_mapping[cleaned_type],
-                        type_snake_case=FhirXmlSchemaParser.camel_to_snake(cleaned_type)
-                        if cleaned_type not in FhirXmlSchemaParser.cleaned_type_mapping
-                        else FhirXmlSchemaParser.camel_to_snake(cleaned_type),
+                        cleaned_type=FhirXmlSchemaParser.cleaned_type_mapping.get(cleaned_type, cleaned_type),
+                        type_snake_case=FhirXmlSchemaParser.camel_to_snake(
+                            FhirXmlSchemaParser.cleaned_type_mapping.get(cleaned_type, cleaned_type)
+                        ),
```

Comment:
this should directly be
`type_snake_case=FhirXmlSchemaParser.camel_to_snake(cleaned_type)`

Reply:
No, The error at sonarqube is saying that the value is same no matter what. Here we seem to have changed the logic that we are passing `FhirXmlSchemaParser.cleaned_type_mapping.get(cleaned_type, cleaned_type)` as argument to `FhirXmlSchemaParser.camel_to_snake`. Previously it we were only passing `cleaned_type` as arg.